src/map.cpp

```diff
@@ -6995,6 +7094,8 @@ void map::actualize( const int gridx, const int gridy, const int gridz )
 
             restock_fruits( pnt, time_since_last_actualize );
 
+            produce_sap( pnt, time_since_last_actualize );
```

Comment:
Very minor problem: sap should be produced after rad scorching, to avoid maples getting scorched but still managing to produce the full stack of sap before it happens.